cls/SourceControl/Git/Settings.cls

```diff
@@ -182,8 +182,9 @@ ClassMethod Configure() As %Boolean [ CodeMode = objectgenerator ]
             do %code.WriteLine(" set list(1) = ""DEVELOPMENT""")
             do %code.WriteLine(" set list(2) = ""TEST""")
             do %code.WriteLine(" set list(3) = ""LIVE""")
-            do %code.WriteLine(" set list(4) = """"")
-            do %code.WriteLine(" set value = """"")
+            do %code.WriteLine(" set list(4) = ""FAILOVER""")
+            do %code.WriteLine(" set list(5) = """"")
+            do %code.WriteLine(" set value = inst.property")
```

Comment:
This is wrong - should be `do %code.WriteLine(" set value = inst."_property)` - although actually, shouldn't be needed at all because we do that on line 162.

Reply:
@isc-etamarch we overlapped on this - you were fixing as I commented, but my point still stands that we shouldn't need to set value here at all.